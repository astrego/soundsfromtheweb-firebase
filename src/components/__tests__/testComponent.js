import React from 'react';
import renderer from 'react-test-renderer';
import TestComponent from '../testComponent';
describe('testComponent', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<TestComponent />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
