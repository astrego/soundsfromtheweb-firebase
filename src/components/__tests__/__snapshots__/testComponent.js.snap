// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testComponent renders correctly 1`] = `
<div>
  <h1>
    TEST
  </h1>
</div>
`;
